### PR TITLE
Created wishlist page and authentication

### PR DESCRIPTION
**Pull request by aryanandarnav13**

### CODE REVIEW

File: wishlist.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <div class="nav-title">CosmosKart</div>
+      </div>
+      <div class="nav-btn">
+        <label for="nav-check">
+          <span></span>
+          <span></span>
+          <span></span>
+        </label>
+      </div>
+      <div class="nav-links">
```
Comment (keshavkumar-in) [inline wishlist.html:25]: you are not consitant with your namin convention, you have used multiple types of naming in these

**Review by keshavkumar-in — APPROVED** [review]
You can improve your class naming convention

File: wishlist.html (hunk 1 above)
Comment (aryanandarnav13) [reply wishlist.html:25]: Noted, I'll correct

**Review by sks-21 — APPROVED** [review]
Good to merge

File: signup.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            <div class="card">
+                <form action="">
+                    <label for="">Email</label>
+                    <br>
+                    <input type="email">
+                    <br><br>
+                    <label for="">First Name</label>
+                    <br>
+                    <input type="text">
+                    <br><br>
```
Comment (arjun-computer-geek) [inline signup.html:27]: this is bad things to use lots of br tag

**Pull request merged**